deploy/deploy-consumer.yaml

```diff
@@ -22,16 +22,13 @@ spec:
         app: rabbitmq-consumer
     spec:
       containers:
-      - name: rabbitmq-consumer
-        image: jeffhollan/rabbitmq-client:dev
-        imagePullPolicy: Always
-        command:
-          - receive
-        args:
-          - 'amqp://user:PASSWORD@rabbitmq.default.svc.cluster.local:5672'
-        envFrom:
```

Comment:
If you remove this, then the consumer won't work anymore AFAIK

Reply:
Yes, you are right

Reply:
I would like to correct myself, this won't cause any problem. Since consumer is getting url from args